environment/src/main/java/com/sequenceiq/environment/environment/dto/EnvironmentDtoConverter.java

```diff
@@ -45,13 +51,27 @@ public EnvironmentDto environmentToDto(Environment environment) {
                 .withStatusReason(environment.getStatusReason())
                 .withTunnel(environment.getTunnel())
                 .withSecurityAccess(environmentToSecurityAccessDto(environment));
+        if (environment.getCloudPlatform().equals(CloudPlatform.AWS.name())) {
+            builder.withAws(getParameterAs(environment, AwsEnvironmentParamsDto.class));
+        }
         if (environment.getNetwork() != null) {
             builder.withNetwork(environmentNetworkConverterMap.get(CloudPlatform.valueOf(environment.getCloudPlatform()))
                     .convertToDto(environment.getNetwork()));
         }
         return builder.build();
     }
 
+    private <T> T getParameterAs(Environment environment, Class<T> clss) {
```

Comment:
It's not a very good solution to handle jsons, do it like we did with `EnvironmentTelemetry`.

Reply:
Or what's even better: don't use json at all, for this single field. It's overkill and unnecessary. Use the relational database like it's designed to be used.

Reply:
reworked